plugins/package-managers/maven/src/test/kotlin/utils/MavenSupportTest.kt

```diff
@@ -17,17 +17,35 @@
  * License-Filename: LICENSE
  */
 
+@file:Suppress("DEPRECATION") // For deprecated ArtifactRepository interface.
```

Comment:
For narrower scoping, please move this (without `file:`) to before line 232.

Reply:
This does not seem to work. The build then fails with:
```
> Task :plugins:package-managers:maven-package-manager:compileTestKotlin FAILED
e: warnings found and -Werror specified
w: file:///home/heo1wa3/projects/ort/plugins/package-managers/maven/src/test/kotlin/utils/MavenSupportTest.kt:32:8 'interface ArtifactRepository : Any' is deprecated. Deprecated in Java.
```

Reply:
Ah, that's because the import itself is also flagged. Please solve that by using the FQN in the tests, like

    val artifactRepository = mockk<org.apache.maven.artifact.repository.ArtifactRepository>()

and removing the import.